<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserLogin</class>
 <widget class="QWidget" name="UserLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Login</string>
  </property>
  <widget class="QPushButton" name="pushButtonLogin">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Log In</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEmailAddress">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>241</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>Enter email address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>241</width>
     <height>30</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>Enter password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>User Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/user-login.svg</normaloff>../icons/user-login.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
